src/main/java/de/retest/recheck/persistence/RecheckTestReportUtil.java

```diff
@@ -25,7 +24,7 @@ private RecheckTestReportUtil() {
 	}
 
 	public static void persist( final SuiteReplayResult suite, final File file ) {
-		logger.info( "Persisting test report to file '{}'.", FileUtil.canonicalPathQuietly( file ) );
+		logger.info( "Persisting test report to file '{}'.", file.getAbsoluteFile().toURI() );
```

Comment:
Good idea, but wouldn't an URL fit better as we specifically want a location?

```suggestion
		logger.info( "Persisting test report to file '{}'.", file.getAbsoluteFile().toURL() );
```

Reply:
`toURL` on file is deprecated. `toURI().toURL()` throws a checked `MalformedUrlException` and should produce the same result. Thus I would keep it as is...